tests/testthat/test-convert_es.R

```diff
@@ -0,0 +1,419 @@
+#' Loads in a .RData file that has pre-made variables (from the vignette) and tests them
+#' with manually created data.frames.
+#'
+#' TODO: Fine-tune the tolerance values
+
+context("convert_es - testing vignette examples")
+
+library(psychmeta)
+
+# output_es = "r" ---------------------------------------------------------
+
+test_that("d to r conversion", {
+
+  # Creating expected
+  expected_convert_es <- data.frame(
+    r = c(0.4472136),
+    n_effective = c(100),
```

Comment:
Update the column names in your expected data frame to match the revised consistent column names in convert_es()